modules/ROOT/partials/plugin-files/plugin-file-list-mergetags.adoc

```diff
@@ -0,0 +1,3 @@
+....
+./plugins/mergetags/plugin.js
```

Comment:
This is an incomplete list... it's missing language files at the very least.

Reply:
lang files are added to the list